src/components/Link/index.js

```diff
@@ -0,0 +1,11 @@
+import PropTypes from "prop-types";
+import React from "react";
+
```

Comment:
Should we import `GatsbyLink from "gatsby-link"` here and use the `GatsbyLink` for this component? Then we can import this into the rest of the design system.

Reply:
It's a bit tricky as I believe `<GatsbyLink>` can _only be used_ for internal links.

So, we're using `<Link>` within the examples and `<GatsbyLink>` within the design system itself. 

Fortunately, because we're using the default device/browser styling, both visually look the same.